packages/component/src/Utils/isSSML.ts

```diff
@@ -0,0 +1,6 @@
+const SPEAK_TAG_PATTERN = /^\s*<speak(\s|\/?>)/u;
+const XML_PROLOG_PATTERN = /^\s*<\?xml\s/u;
```

Comment:
For future-proofing, regex should have a description comment. I also like the idea of linking to a set of regex tests on, for example, regex101. Or maybe just add to our own tests? This also may help if the SSML protocol changes in the future, right?

1. English-readable description, including purpose (Speech, etc)
2. Text comparison examples (via link? path to Web Chat regex tests?)
3.

Reply:
This file is removed. It was borrowed from DLSpeech package.